deploy/docker/start.sh

```diff
@@ -1,47 +1,42 @@
 #!/bin/sh
-set -e
+set -e -u
 
 emqx_exit(){
-    # tail erlang.log.*
-    erlang_log=$(echo $(ls -t /opt/emqx/log/erlang.log.*) | awk '{print $1}')
-    num=$(sed -n -e '/LOGGING STARTED/=' ${erlang_log} | tail -1)
-    tail -n +$((num-2)) ${erlang_log}
+    # tail emqx.log.*
+    emqx_log=$(echo $(ls -t /opt/emqx/log/emqx.log.*) | awk '{print $1}')
+    num=$(sed -n -e '/LOGGING STARTED/=' ${emqx_log} | tail -1)
+    tail -n +$((num-2)) ${emqx_log}
 
     echo "['$(date -u +"%Y-%m-%dT%H:%M:%SZ")']:emqx exit abnormally"
     exit 1
 }
 
 ## EMQ Main script
 
-# Start and run emqx, and when emqx crashed, this container will stop
+# Set log.rotation=off in etc/emqx.conf
+sed -i -r 's/log.rotation = (.*)/log.rotation = off/1' /opt/emqx/etc/emqx.conf
```

Comment:
question, is this going to stop the rotation hence producing `emqx.log` only ?
if yes, why the `emqx.log.*` pattern at line 6 ?

Reply:
> question, is this going to stop the rotation hence producing `emqx.log` only ?
Yes, It will only generate `emqx.log`

> if yes, why the `emqx.log.*` pattern at line 6 ?
This is my mistake, I just modified it